oauth

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
+
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
```

Comment:
this file will be downloaded at `./vendor/bin` as per composer standards (just guessing), are you sure this will work?